blog/posts/announcing-melange-5.md

```diff
@@ -0,0 +1,285 @@
+`@mel.send.pipe`. The only real reason why 2 constructs existed to do the same
+was to support two alternatives for chaining them in OCaml:
+[pipe-first](https://melange.re/v5.0.0/language-concepts.html#pipe-first) and
+[pipe-last](https://melange.re/v5.0.0/language-concepts.html#pipe-last). But
+this always felt like an afterthought, and code using `@mel.send.pipe` never
+felt intuitive to look at (e.g. in `external say: unit [@mel.send.pipe: t]`,
+one had to mentally place the `t` before `unit`, since the real signature is `t
+-> unit`).
+
+In Melange 5, we wanted to remove this weird split and further reduce the
+cognitive overhead of writing bindings to call JavaScript methods on an object
+or instance.
+
+We're introducing a way to mark the "self" instance argument with
+`@mel.this` and recommending only the use of `@mel.send` going forward.
+Starting from this release, `@mel.send.pipe` has been deprecated, and will be
+removed in the next major release of Melange. Here's an example:
+
+```js
+[@mel.send]
+external push: (~value: 'a=?, [@mel.this] array('a)) => unit = "push";
+
+let () = push([||], ~value=3);
+```
+
+The code above marks the `array('a)` argument as the instance to call the
+`push` method, which produces the following JavaScript:
+
+```js
+[].push(3);
+```
+
+Besides being more versatile, having an explicit marker with `@mel.this` is
+also more visually intuitive: when scanning Melange code containing `external`
+bindings, it becomes easier to spot which is the "this" argument. This feature
+is 100% backwards compatible with `@mel.send`: in the absence of `@mel.this`,
+the instance argument defaults to the first one declared in the signature, as
+previously supported.
+
+## Additional quality of life improvements
+
+### OCaml 5.3 Compatibility / Stdlib Upgrade
+
+Since [Melange's
+inception](https://anmonteiro.com/2021/03/on-ocaml-and-the-js-platform/), one
+of its goals has been to keep it up to date with the latest OCaml releases.
+[This
+release](https://github.com/melange-re/melange/releases/download/5.0.1-53/melange-5.0.1-53.tbz)
+brings Melange up to speed with OCaml 5.3, including upgrades to the `Stdlib`
+library as well. We're also releasing Melange 5 for OCaml
+[4.14](https://github.com/melange-re/melange/releases/download/5.0.1-414/melange-5.0.1-414.tbz),
+[5.1](https://github.com/melange-re/melange/releases/download/5.0.1-51/melange-5.0.1-51.tbz)
+and
+[5.2](https://github.com/melange-re/melange/releases/download/5.0.1-52/melange-5.0.1-52.tbz).
+
+### Melange runtime NPM packages
+
+Starting from this release, we're shipping NPM packages with the precompiled
+Melange runtime. This feature, requested by a few users in
+[melange#620](https://github.com/melange-re/melange/issues/620) allows to use
+Melange without compiling its own runtime and stdlib (essentially, in
+combination with `(emit_stdlib false)` in `(melange.emit ..)`).
+
+This can be useful in monorepos that compile multiple Melange applications but,
```

Comment:
This use case is still unclear to me. Assuming a single opam switch, why would one want to add npm overhead (and potential mismatch of melange versions definitions between opam and npm)?